docs/configdoc.md

```diff
@@ -44,56 +44,65 @@ environment: PROD
 |BpfMapDefaultPath|`"/sys/fs/bpf"`|The base pin path for eBPF maps| Yes |
 
 ## [kf-repo]
-| FieldName     | Example       | Description     | Required        |
-| ------------- | ------------- | --------------- | --------------- |
-|url| `"http://localhost:8000/"`|Default repository from which to download eBPF packages| No |
+| FieldName     | Example       | Description     | Required |
```

Comment:
Please change heading "Example" to "Default" to match the format below.  We should do this for all sections.